### DISCUSSION TITLE
Want to contribute on [ [1] An Integrated CONTROL-CORE experience]

### DISCUSSION THREAD

**Discussion by Sujan804** [ControlCore-Project/concore-editor#63] [General]
Hi Folks,
I am Sujan Ahmed. A final year CS undergrad Student. Working with JavaScript, React, Nodejs, Express, and python. I have read the details of the project and have been observing the code base of the project for several days. I think the project is interesting for me to work on.
I am highly interested in getting mentored and to contribute this project. It would be great if I am notified about the next step.

**Comment by pradeeban** [comment]
Hi and welcome, @Sujan804.

Since you have started exploring the project ideas of concore-editor, I recommend you go through the GitHub repository and have an attempt at some of the listed bug reports. #62 can be a good first issue to attempt.

**Comment by JaeAeich** [comment]
Dear GSoC mentors,

I am Javed Habib, a Computer Science and Engineering major from the Indian Institute of Technology (IIT) Bhilai, graduating in 2025. I am writing to request your guidance on the GSoC project "An Integrated CONTROL-CORE experience" which I am excited to contribute to.

I have successfully set up the development environment and am ready to start working on the project. I would appreciate any guidance and support  on how I can get started with the project and contribute effectively. Specifically, I would like to know where I can start, and any specific areas I should focus on.

Thank you for considering my request, and I look forward to hearing back from you soon.

Sincerely,
Javed Habib.
